pkg/security/tpm.go

```diff
@@ -0,0 +1,48 @@
+package security
+
+import (
+	"crypto/sha1"
+
+	tspi "github.com/google/go-tpm/tpm"
+)
+
+const (
+	tpmDefaultDevice string = "/dev/tpm0"
+)
+
+// ReadPcrTPM1 reads the PCR for the given
+// index
+func ReadPcrTPM1(pcr uint32) ([]byte, error) {
```

Comment:
All the functions below look good to me. I also have a suggestion for the interface.

These functions work with TPM 1.2 only. Since I expect we will also have functions that deal with TPM 2 as well, what about making two `struct`s, one for TPM1.2 and one for TPM2.0 instead of top-level functions? Something like this:

```
type TPM1 struct {
    Device string
}

func NewTPM1() {
    return &TPM1{Device: DefaultTPMDevice}
}

func (t1 TPM1) ReadPcr(pcr uint21) ([]byte, error) {
    ...
}

func (t1 *TPM1) Measure(pcr uint21, data []byte) error {
    ...
}
```

The usage would become something like:
```
tpm := NewTPM1()
if err := tpm.Measure(...); err != nil {
    ...
}
```

This will also simplify the implementation of `TPM2`. Eventually we could even make an interface for shared features of both TPM versions. What do you think?